app/models/organisation.rb

```diff
@@ -5,4 +5,6 @@ class Organisation < ApplicationRecord
   has_many :users
 
   has_many :mou_signatures
+
+  scope :with_users, -> { where(User.where("organisation_id = organisations.id").arel.exists) }
```

Comment:
Does orgs have a default sort? I'm wondering if it would be worth adding a sort here to make sure its in some sort of order

Reply:
🤔  does it matter if the user is an editor or trial account?